Create datebase

yuya-oki opened the pull request:
What
READEME.mdにデータベース設計を行った
WHY
送信されたものを保存するため。

Commit:
Create datebase

hiroki5123 commented on chat-space/README.md:
```diff
@@ -1,24 +1,54 @@
+|Column|Type|Option|
+|------|----|------|
+|user_id|integer|null: false, foreign_key: true|
+|user_name|string|null: false, foreign_key: true|
```
nameでいいと思います

hiroki5123 commented on chat-space/README.md:
```diff
@@ -1,24 +1,54 @@
+## userテーブル
+|Column|Type|Option|
+|------|----|------|
+|user_id|integer|null: false, foreign_key: true|
```
これは必要ありません

hiroki5123 commented on chat-space/README.md:
```diff
@@ -1,24 +1,54 @@
+|------|----|------|
+|user_id|integer|null: false, foreign_key: true|
+|user_name|string|null: false, foreign_key: true|
+|user_email|string|null: false, add_index, unique: true|
```
deviseで作成するカラムということを頭に置いておいてください。

Commit:
change file

WHAT
user_name->name
user_idを消した
add_indexを消した

WHY
user_nameはnameでも理解できるから
user_id,add_indexは必要ないため
